src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1534,6 +1541,9 @@ def execute(my):
         category = my.kwargs.get("category")
         keywords = my.kwargs.get("keywords")
         update_process = my.kwargs.get("update_process")
+        hide_path_keywords = my.kwargs.get("hide_path_keywords")
+        if hide_path_keywords:
+            hide_path_keywords = hide_path_keywords.split(",")
```

Comment:
sometimes ppl word1, word2.. so do a strip for the results like this
hide_path_keywords = [x.strip() for x in hide_path_keywords]